.github/ISSUE_TEMPLATE/1-remote-rendering-problem.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Expose as a link to arr github issue:
https://github.com/Azure/azure-remote-rendering/issues/new?assignees=&labels=rendering,arrt&template=1-remote-rendering-problem.md&title=%5BRemote+rendering+problem%5D

See &labels, both rendering and arrt need to be created first in ARR repo via https://github.com/Azure/azure-remote-rendering/issues/labels